### DISCUSSION TITLE
wired table 识别时，如果一个图片上有多个表格，怎样返回每张表格分开的html 和表格坐标？

### DISCUSSION THREAD

**Discussion by nissansz** [RapidAI/TableStructureRec#46] [Q&A]
wired table 识别时，如果一个图片上有多个表格，怎样返回每张表格分开的html 和表格坐标？

**Comment by Joker1212** [comment]
这个仓库做不到哈，这是表格检测该做到事情，目前该仓库只处理单表格且无明显旋转透视的识别

**Reply by nissansz** [reply]
透视的表格，有什么办法纠偏吗？

**Reply by Joker1212** [reply]
可以进行边缘识别或者找专门的模型来做哈，这个仓库目前没有这块，后面可能会补充
